.github/workflows/go.yml

```diff
@@ -18,14 +18,20 @@ jobs:
       with:
         go-version: stable
 
-    - name: Run tests in maven
-      working-directory: ./util/maven
-      run: go test ./...
+    - name: Build examples
+      working-directory: examples/go/
+      run: |
```

Comment:
This can be a one-liner with find:

    run: find . -name 'go.mod' -execdir go build \;